tests/test-sources/plugins/by-name/cmp-tabnine/default.nix

```
@@ -0,0 +1,49 @@
+{ lib, pkgs, ... }:
+let
+  platform = pkgs.stdenv.hostPlatform;
+
+  # tabnine is not available on aarch64-linux.
+  doRun = !(platform.isLinux && platform.isAarch64);
+in
+lib.optionalAttrs doRun {
```

This is the same, right?

```suggestion
lib.optionalAttrs (platform.system != "aarch64-linux") {
```

---

Yes, considered to be less robust (string comparison ban be wrong and fail silently).